activerecord/lib/active_record/connection_adapters/sqlite3/column.rb

```diff
@@ -1,15 +1,18 @@
 # frozen_string_literal: true
 
+require "active_support/core_ext/object/blank"
```

Comment:
`object/blank` is already required in all frameworks by default:
https://github.com/rails/rails/blob/162be9f2196a02c72b5b15944352a33b94e47b6e/activerecord/lib/active_record.rb#L27
https://github.com/rails/rails/blob/162be9f2196a02c72b5b15944352a33b94e47b6e/activesupport/lib/active_support/rails.rb#L14

Reply:
Interesting. I copied this from the PostgreSQL adapter, so I will remove it from there as well.